src/ebonite/ext/flask/helpers.py

```diff
@@ -1,26 +1,27 @@
 import os
 
+from ebonite.build.docker_objects import DockerImage
 from ebonite.build.helpers import build_model_docker
 from ebonite.core.objects import core
 from ebonite.ext.flask.server import FlaskServer
 
 TEMPLATES_DIR = 'build_templates'
 
 
-def build_model_flask_docker(image_name: str, model: 'core.Model', image_tag='latest', force_overwrite=False, debug=False):
+def build_model_flask_docker(image_params: DockerImage, model: 'core.Model', force_overwrite=False, debug=False) \
```

Comment:
🤔 helpers are supposed to be used without additional imports, like you dont need to import Model to use Model.create, you can use helper method create_model. So I think this should accept `image_name` string and other arguments as kwargs.
And while I wrote this, it got me thinking. Maybe we should allow users to build images without client and repos. But if you use client, they will be saved to repos and so on. 
And one more thing, there must be a way to tell client that you want to build 'flask' image, when you call build_image method

Reply:
Fixed to support both params/name arguments (btw without kwargs, just `Union` argument).

Reply:
Building images without client and repos is the main point of this method for me. Client hust wraps it with persistence (repo) call. Thus I treat this feature as already available.

Reply:
fixed to use flask uwsgi by default (if server is not given)